docs/user_guide/ttl.md

```diff
@@ -14,10 +14,32 @@ store-version, after the push finishes, but before swapping the read traffic ove
 replay phase where the last N seconds of buffered real-time data is replayed. The replayed data gets overlaid on top of
 the data from the Full Push.
 
-This can be leveraged in order to achieve a TTL behavior. By scheduling periodic empty pushes and configuring how far 
-back to replay the real-time data (N), one can control the TTL parameters. The time to live is defined by N, which acts 
-as a "minimum TTL", while the "maximum TTL" is N + delay between each empty push. For example, if you schedule a daily 
-empty push, and N = 6 days, then the oldest data in your store will be at least 6 days old, and at most 7 days old.
+This can be leveraged in order to achieve a TTL behavior. Currently, there are two options that leverages this mechanism. 
+
+### Empty push
+By scheduling periodic empty pushes and configuring how far back to replay the real-time data (N), one can control the 
+TTL parameters. The time to live is defined by N, which acts as a "minimum TTL", while the "maximum TTL" is N + delay 
+between each empty push. For example, if you schedule a daily empty push, and N = 6 days, then the oldest data in your 
+store will be at least 6 days old, and at most 7 days old.
+
+The N is as known as the store-level rewind time, that's a part of hybrid store configuration.
+
+### [Experimental] Repush with TTL
+Similarly to empty push, the repush with TTL can also be configured and scheduled periodically to achieve the TTL behavior.
+The major differences between this and empty push are original data source and how real-time buffer is replayed.
+
+The empty push literally has no original data, i.e. starts from scratch, and then replay real-time buffer from past N days,
+whereas the repush scans through entries in the current version, as its original data, and evict stale records based on 
+the write timestamp, and finally replays the real-time buffer from past M days. The M is known as version-level rewind time,
```

Comment:
I must have missed some detail here, but why do we have both M and N? And why would M be "normally smaller than the N"? What if it isn't? Is that a problem?

Reply:
> why do we have both M and N?

For repush job, it has a default 24 hrs for`rewindTimeInSecondsOverride`. So in this case, both M and N exist but repush would use the M. 

>  And why would M be "normally smaller than the N"? What if it isn't? Is that a problem?

I should be careful about the wording here. The statement of "M is normally smaller than N" is coming from heuristic. Nothing wrong would happen if M is larger than N. Will remove this.